staker/staker.go

```diff
@@ -453,7 +453,7 @@ func (s *Staker) Start(ctxIn context.Context) {
 			backoff = time.Minute
 			log.Error("error acting as staker", "err", err)
 		} else {
-			log.Warn("error acting as staker", "err", err)
+			log.Warn("block validation is still pending")
```

Comment:
There's a lot of cases where we could fail to act as staker for a different reason and would still want to log the error. What I'd do is instead change the error itself about having the wrong module root to just "block validation is still pending" if the list of WASM module roots we have in the block validator is empty.